[Test:] `azurerm_cdn_profile`/`azurerm_cdn_endpoint`/`azurerm_frontdoor` - fix tests due to deprecated `sku` values by RP (#28908)
hashicorp/terraform-provider-azurerm: website/docs/r/cdn_endpoint.html.markdown

A CDN Endpoint is the entity within a CDN Profile containing configuration information regarding caching behaviours and origins. The CDN Endpoint is exposed using the URL format `<endpointname>.azureedge.net`.

> !> **Note:** Azure is rolling out a breaking change on Friday 9th April 2021 which may cause issues with the CDN/FrontDoor resources. [More information is available in this GitHub issue](https://github.com/hashicorp/terraform-provider-azurerm/issues/11231) - however unfortunately this may necessitate a breaking change to the CDN and FrontDoor resources, more information will be posted [in the GitHub issue](https://github.com/hashicorp/terraform-provider-azurerm/issues/11231) as the necessary changes are identified.
> 
> !> **Note:** The CDN services from Edgio(formerly Verizon) will be shut down on or before 15 January 2025. All Azure CDN from Edgio customers must migrate their workloads out of Azure CDN from Edgio before 15 January 2025.
> 
> !> **Note:** The CDN services from Akamai support will be removed on 31 October 2023. All Azure CDN from Akamai customers must migrate their workloads out of Azure CDN from Akamai before 31 October 2023.

Reviewer: I think these should be in the past tense for events that have already happened?

```suggestion
!> **Note:** Azure rolled out a breaking change on Friday 9th April 2021 which may cause issues with the CDN/FrontDoor resources. [More information is available in this GitHub issue](https://github.com/hashicorp/terraform-provider-azurerm/issues/11231) - unfortunately this may necessitate a breaking change to the CDN and FrontDoor resources, more information will be posted [in the GitHub issue](https://github.com/hashicorp/terraform-provider-azurerm/issues/11231) as the necessary changes are identified.

!> **Note:** The CDN services from Edgio(formerly Verizon) was shut down on 15 January 2025 and is no longer available.

!> **Note:** Support for CDN services from Akamai was removed on 31 October 2023.
```
Trimmed the last note as (correct me if I'm wrong) there are no Akamai deployed FD's anywhere now?
